staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -1038,6 +1038,11 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 		handler = genericapifilters.WithTracing(handler, c.TracerProvider)
 	}
 	handler = genericapifilters.WithLatencyTrackers(handler)
+	// WithRoutine will execute future handlers in a separate goroutine and serving
+	// handler in current goroutine to minimize the stack memory usage.
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServingWithRoutine) {
```

Comment:
Please also add a comment that this has to be put after WithPanicRecovery (as it is now), but to ensure that someone will not change it later.

Also please squash the commits.

Reply:
Added comment and squashed.